scripts/azureml-assets/azureml/assets/release/asset_publish.py

```diff
@@ -123,12 +124,19 @@ def _str2bool(v: str) -> bool:
                                                              asset_name=asset.name,
                                                              version=final_version)
         # Handle specific asset types
-        if asset.type in [assets.AssetType.COMPONENT, assets.AssetType.ENVIRONMENT]:
+        if asset.type in [assets.AssetType.COMPONENT, assets.AssetType.ENVIRONMENT, assets.AssetType.MODEL]:
```

Comment:
since model is already being handled differently now, can we take it out and have another ifelse?